xbmc/cores/VideoPlayer/DVDCodecs/Video/AMLCodec.cpp

```diff
@@ -1754,7 +1754,7 @@ bool CAMLCodec::OpenDecoder(CDVDStreamInfo &hints)
 
   CLog::Log(LOGNOTICE, "CAMLCodec::OpenDecoder - using V4L2 pts format: %s", m_ptsIs64us ? "64Bit":"32Bit");
 
-  m_ptsOverflow = (sizeof(long) == 8) ? 0 : INT64_0;
+  m_ptsOverflow = 0;
```

Comment:
For 64bit AML lib no PTSOverflow is reqired, are yousure that the logic change still works?
This change disables ptsOverflow for 32 bit AML libs now. My gut feeling is that the author has not fully understood the previous logic and changed by try.
But I'm Ok with all the changes here, because the AML lib support will hopefully die soon (V4L2)

Reply:
It will still work as there is a check for arm/aarch64 in https://github.com/xbmc/xbmc/pull/15859/files#diff-4e99a88003816cb71e489e8a0db42345R1936